src/ImageSharp/PixelFormats/PixelOperations{TPixel}.Generated.cs

```diff
@@ -0,0 +1,302 @@
+        /// <param name="sourceBytes">The <see cref="Span{T}"/> to the source bytes.</param>
+        /// <param name="destPixels">The <see cref="Span{T}"/> to the destination pixels.</param>
+        /// <param name="count">The number of pixels to convert.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal void PackFromBgra32Bytes(Span<byte> sourceBytes, Span<TPixel> destPixels, int count)
+        {
+            this.PackFromBgra32(sourceBytes.NonPortableCast<byte, Bgra32>(), destPixels, count);
+        }
+		
+		/// <summary>
+        /// Converts 'count' pixels in 'sourcePixels` span to a span of <see cref="Bgra32"/>-s.
+        /// Bulk version of <see cref="IPixel.ToBgra32(ref Bgra32)"/>.
+        /// </summary>
+        /// <param name="sourcePixels">The span of source pixels</param>
+        /// <param name="dest">The destination span of <see cref="Bgra32"/> data.</param>
+        /// <param name="count">The number of pixels to convert.</param>
+        internal virtual void ToBgra32(Span<TPixel> sourcePixels, Span<Bgra32> dest, int count)
+        {
+            GuardSpans(sourcePixels, nameof(sourcePixels), dest, nameof(dest), count);
+
+            ref TPixel sourceBaseRef = ref sourcePixels.DangerousGetPinnableReference();
+            ref Bgra32 destBaseRef = ref dest.DangerousGetPinnableReference();
+
+            for (int i = 0; i < count; i++)
+            {
+                ref TPixel sp = ref Unsafe.Add(ref sourceBaseRef, i);
+                ref Bgra32 dp = ref Unsafe.Add(ref destBaseRef, i);
+                sp.ToBgra32(ref dp);
+            }
+        }
+
+		/// <summary>
+        /// A helper for <see cref="ToBgra32(Span{TPixel}, Span{Bgra32}, int)"/> that expects a byte span as destination.
+        /// The layout of the data in 'destBytes' must be compatible with <see cref="Bgra32"/> layout.
+        /// </summary>
+        /// <param name="sourceColors">The <see cref="Span{T}"/> to the source colors.</param>
+        /// <param name="destBytes">The <see cref="Span{T}"/> to the destination bytes.</param>
+        /// <param name="count">The number of pixels to convert.</param>
+		[MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal void ToBgra32Bytes(Span<TPixel> sourceColors, Span<byte> destBytes, int count)
+        {
+            this.ToBgra32(sourceColors, destBytes.NonPortableCast<byte, Bgra32>(), count);
+        }
+		
+		/// <summary>
+        /// Converts 'count' elements in 'source` span of <see cref="Rgb24"/> data to a span of <typeparamref name="TPixel"/>-s.
+        /// </summary>
+        /// <param name="source">The source <see cref="Span{T}"/> of <see cref="Rgb24"/> data.</param>
+        /// <param name="destPixels">The <see cref="Span{T}"/> to the destination pixels.</param>
+        /// <param name="count">The number of pixels to convert.</param>
+        internal virtual void PackFromRgb24(Span<Rgb24> source, Span<TPixel> destPixels, int count)
```

Comment:
It's a bit more complicated because of the `Span<byte>` overloads (`PackTo/PackFrom****Bytes`), where `Span<byte>` could be both source and destination. Because of them, the simplest solution was to specify the pixel count explicitly everywhere.

I think I can manage it without `count`, but need to have a deeper look at the call sites.